WcaOnRails/app/controllers/competitions_controller.rb

```diff
@@ -227,6 +227,8 @@ def post_results
     end
     post = Post.new(title: title, body: body, author: current_user, world_readable: true)
     post_post(post)
+    comp.results_posted = Time.now
+    comp.save!
```

Comment:
I find `post_post(post)' pretty silly (not so bad) and cryptic (bad!) code.

I'd change the method to

    def create_post_and_redirect(post_attrs)
        @post = Post.new(post_attrs)
        ...